### ISSUE TITLE
READING I@C BNo080 sensor

### ISSUE THREAD

**Issue by nafizz1410** [fm4dd/pi-bno080#4]
Hi can anyone please teach me how to read i2c bn o080 sensor pleasee

**Comment by nafizz1410** [comment]
Hi sir can i know how to read the snsor reading?

**Comment by nafizz1410** [comment]
Hi @sezeryalcin and @fm4dd i really need help in reading this sensor with my rpi. Can you share with me the code? please I am only interested to read the gyro module. I have tried to read it since last two weeks.

Thanl you in advance

**Comment by sezeryalcin** [comment]
@nafizz1410 There is no usable RPI code for BNO080 in public. There is some out there that doesn't work and some are outdated. Arduino code (i2c) is out there and working fine.
You can try BNO080 in serial/UART mode and experiment there. However there is no useful ready-made code for that either. I am coding something to get euler angles over UART in RPI.

**Comment by nafizz1410** [comment]
@sezeryalcin ahve you found the code to get euler angle over UART?

**Comment by sezeryalcin** [comment]
Yes, we can read in UART-RVC mode which needs PS0 jumper to be soldered. We may release the sofware which is script based anyway when complete.
